composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "sof3/libasynql",
     "type": "library",
     "require": {
-        "pocketmine/pocketmine-mp": "^4.0.0"
+        "pocketmine/pocketmine-mp": "dev-major-next"
```

Comment:
`^5.0.0-BETA4` would be better for this.

Reply:
we're not tagging the pm5 branch though, why do we need to lock in to a version?

Reply:
does `^5.0.0` work for any beta version? Looks like `^5.0.0` would be better for future.

Reply:
Not sure. You're welcome to try it.

Reply:
Looks like it doesn't work.

Reply:
^5.0.0-BETA4 will accept any 5.x.y release anyway, so this is fine.